test/mail_ssl_manual.bats

```diff
@@ -101,11 +101,9 @@ function teardown_file() {
 @test "checking ssl: manual cert changes are picked up by check-for-changes" {
     printf 'someThingsChangedHere' \
       >>"$(pwd)/test/test-files/ssl/${DOMAIN_SSL_MANUAL}/with_ca/ecdsa/key.ecdsa.pem"
-    sleep 15
 
-    run docker exec mail_manual_ssl /bin/bash -c "supervisorctl tail -3000 changedetector"
-    assert_output --partial 'Change detected'
-    assert_output --partial 'Manual certificates have changed'
+    run timeout 15 docker exec mail_manual_ssl bash -c "tail -F /var/log/supervisor/changedetector.log | sed '/Manual certificates have changed/ q'"
```

Comment:
Is this a way to get the full log? I think I had difficulty with tests I wrote that could properly retrieve that, so I relied on the supervisor tail command instead which is problematic..

---

Just to confirm I understand the command, it will tail the log file, piping the output to sed until the log is updated with the line sed is looking for, then sed quits with success, or the timeout results in failure?

Reply:
> Is this a way to get the full log?

In this case, yes. The log file does not exist when the test starts. That's why I used `-F`. It will start "tailing" as soon as the file exists.
If you go with the log files directly instead of `supervisorctl tail`, then you have the whole log available. I'll take a look at the other tests, that also uses `supervisorctl tail` and see, if those can also be improved. I wanted to discuss/merge this first, to see if that's a good way to go.

> Just to confirm ...

Correct.